docs/sphinx/manual/hpx-functions.rst

```diff
@@ -0,0 +1,73 @@
+..
+    Copyright (C) 2019 Tapasweni Pathak
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+.. _HPX Functions:
+
+=============
+HPX Functions
+=============
+
+This document describes the prototypes, the allowable arguments of the listed
+HPX functions.
+
+async
+=====
+
+``async`` in C++ allows running the function f asynchronously and returns a
+``hpx::future`` having the result of that function call.
+
+``async`` calls a function f with arguments args as per launch policy. You can
+have async flag set or deferred flag set.
+
+Implementing async functions should have input parameters as
```

Comment:
I don't quite understand this sentence. Do you mean to say that "`hpx::async` takes as input parameters the following...".?